schema/src/main/scala/org/constellation/schema/snapshot/HeightRange.scala

```diff
@@ -6,11 +6,15 @@ import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
 /**
   * Inclusive range of heights
   */
-case class HeightRange(from: Long, to: Long)
+case class HeightRange(from: Long, to: Long) {
+  def nonEmpty: Boolean = !empty
+
+  def empty: Boolean = from > to
```

Comment:
I wonder if the range from 10 to 10 is an empty range? I'm using such range in healthcheck currently to find missing proposals for a given height. Maybe >= ???